src/Illuminate/Contracts/Foundation/Application.php

```diff
@@ -3,7 +3,11 @@
 namespace Illuminate\Contracts\Foundation;
 
 use Illuminate\Contracts\Container\Container;
+use Illuminate\Foundation\Application as App;
```

Comment:
Imports have a runtime performance cost. Better to just write the FQCN in the phpdoc.